posydon/visualization/plot_pop.py

```diff
@@ -245,9 +246,11 @@ def plot_popsyn_over_grid_slice(pop, grid_type, met_Zsun, slices=None, channel=N
         met_indices = np.where(pop.select(where=channel_sel, columns=['metallicity']) == met_Zsun)[0].tolist()
         if 'HMS-HMS' in grid_path:
             slice_3D_var_range = (dq_edges[i],dq_edges[i+1])
-
-            sel = 'index in '+str(met_indices)+' & event == "ZAMS"'
-            data = pop.history.select(where=sel, columns=['S1_mass','S2_mass' ,'orbital_period'])
+            if len(met_indices) == 0:
+                data = pd.DataFrame(columns=['S1_mass','S2_mass' ,'orbital_period'])
+            else:
+                sel = 'index in '+str(met_indices)+' & event == "ZAMS"'
+                data = pop.history.select(where=sel, columns=['S1_mass','S2_mass' ,'orbital_period'])
```

Comment:
Just style: is there a reason for the space behind `'S2_mass'`? Would it be useful to define a common column set once and always use this?

Reply:
You were too quick: now there is a space behind the second comma but not behind the first one, why?